### ISSUE TITLE
Feature Request: Decide AMBE for Pi-Star audio output

### ISSUE THREAD

**Issue by NyxoLyno** [AndyTaylorTweet/Pi-Star_v4_Binaries_Bin#2]
Request use of AMBE decoder tools (https://github.com/g4klx/AMBETools/blob/master/README.md) be added to Pi-Star to allow decoding of audio to be played through Raspberry Pi audio headphone jack. 

It would "appear" as though incorporation of these AMBE decoder tools would allow incoming and outgoing AMBE signals be decided and played. 

Correct me if I'm wrong, please.

**Comment by MW0MWZ** [comment]
Do you have an AMBE3000 Vocoder chip?

**Comment by NyxoLyno** [comment]
I will gladly buy a DVStick 30. https://www.gigaparts.com/dvmega-dvstick-30.html

**Comment by MW0MWZ** [comment]
OK - well some good and bad news - if you look further at what this does, it will transcode captured ambe or imbe data into a wav, it will NOT stream the life stream magically out of the speaker, if it did that - I would already include it.

**Comment by NyxoLyno** [comment]
What about this? https://www.hamradio.com/detail.cfm?pid=H0-017021

**Comment by MW0MWZ** [comment]
So that is an AMBE device for transcoding streams, but as yet I don't think the software has caught up.
There is DummyRepeater - but that is D-Star only.

The closest thing I have seen to what you want is the DV-Cast, it has an onboard SPC and onboard AMBE board, and can be used as either a hotspot or gives you direct decode to audio. Does that help?

**Comment by NyxoLyno** [comment]
Not exactly. So, I have a dual band DVMega, and Pi-Star is capable of receiving and transmitting this packet traffic through a DVMega, Zumspot, etc. What I'm asking is, why can't that same data also be passed through something like a DV-Stick simultaneously for decoding and passed through the audio port? The hardware and software are capable of making such a connection, but Pi-Star would need to be configured to make that data available to both.

**Comment by NyxoLyno** [comment]
This article demonstrates using an RTL-SDR and software to obtain and decide Iridium satellite signals. https://www.rtl-sdr.com/dragonos-decoding-iridium-satellites-with-the-iridium-toolkit-and-an-rtl-sdr/

One of the software tools it uses is an AMBE decoder. There is no AMBE chip in this setup, yet, he is able to decode and play AMBE voice. 

The question is, if this setup can be done with an RTL-SDR, why can't a Raspberry Pi with Zumspot/DVMEGA/MMDVM attached? They receive the signals (like an RTL-SDR), so why not incorporate the AMBE decoder to play the audio?

**Comment by neiroman2k** [comment]
No chance to convert wav to ambe without AMBE3000 stick ?
